app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
Yes , it is only to display in the frontend , cause different timezones may have same offset.